### ISSUE TITLE
Shippo (production) python API doesn't support Python > 3.3

### ISSUE THREAD

**Issue by vsoch** [vsoch/freegenes#24]
I was worried this would be a show stopper, but it looks like there is a release candidate that will support Python > 3.3. For now, I'm installing:

```bash
pip install shippo==2.0.0rc1
```
However this should be updated when the release candidate is an actual release. I'm watching

https://github.com/goshippo/shippo-python-client/issues/45 and the linked issue -

And I'm laughing now because @Koeng101 you are in the discussion on https://github.com/goshippo/shippo-python-client/issues/51 :)

**Comment by Koeng101** [comment]
Yep! I was worried it'd be a show stopper, which is why I had asked back then :)

**Comment by vsoch** [comment]
@Koeng101 when you update the version at some future point, they fixed the bug with adding the shippo token to the client directly (the change was made, but the error message not updated!) See https://github.com/goshippo/shippo-python-client/issues/55#issuecomment-567728843.

**Comment by vsoch** [comment]
hey @Koeng101 ! The updated version was released today for shippo, https://github.com/goshippo/shippo-python-client/issues/45#issuecomment-573189926 hooray!
